crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -113,7 +113,7 @@ def compute_status
           )
         end
         nova = NodeObject.find("roles:nova-controller").first
-        ret[:no_live_migration] = true unless nova["nova"]["use_migration"]
```

Comment:
I'm always confused by the logic after unless. So it's probably better to just `return ret if nova.nil?` as @MaximilianMeister originally suggested